Polyhedron/demo/Polyhedron/Plugins/Point_set/Point_set_normal_estimation_plugin.ui

```diff
@@ -46,7 +46,7 @@
       <item row="0" column="1">
        <widget class="QSpinBox" name="m_pca_neighbors">
         <property name="suffix">
-         <string>Nearest Neighbors</string>
+         <string> Nearest Neighbors</string>
```

Comment:
This way it shows "12 Nearest Neighbors" instead of "12Nearest Neighbors".
How else would you add a space between the number of the SpinBox and the
text ?

2016-07-04 19:34 GMT+02:00 Laurent Rineau <notifications@github.com>:

> In
> Polyhedron/demo/Polyhedron/Plugins/Point_set/Point_set_normal_estimation_plugin.ui
> <https://github.com/CGAL/cgal/pull/1242#discussion_r69481788>:
>
> > @@ -46,7 +46,7 @@
> >        <item row="0" column="1">
> >         <widget class="QSpinBox" name="m_pca_neighbors">
> >          <property name="suffix">
> > -         <string>Nearest Neighbors</string>
> > +         <string> Nearest Neighbors</string>
>
> I am surprised by this modification. Was that on purpose? If it is, what
> is the goal of inserting one blank space of unknown width?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CGAL/cgal/pull/1242/files/04d80304e9579043c8e157dc30094cf394e91c8e#r69481788>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AKyW1ZmsQAy-PWSP46L6x4n4mOORNWLDks5qSUQqgaJpZM4JEMJy>
> .
>